catalog/plugin/catalog-plugin-metacardbackup-filestorage/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,32 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0">
+
+    <cm:managed-service-factory
+            id="metacardFileBackup"
+            factory-pid="org.codice.ddf.catalog.plugin.metacard.backup.storage.filestorage.MetacardBackupFileStorage"
+            interface="org.codice.ddf.catalog.plugin.metacard.backup.storage.api.MetacardBackupStorageProvider">
+        <service-properties>
+            <cm:cm-properties persistent-id="org.codice.ddf.catalog.plugin.metacard.backup.storage.filestorage.MetacardBackupFileStorage" />
+        </service-properties>
+        <cm:managed-component class="org.codice.ddf.catalog.plugin.metacard.backup.storage.filestorage.MetacardBackupFileStorage">
+            <cm:managed-properties persistent-id="" update-strategy="component-managed"
+                                   update-method="refresh"/>
+            <property name="id" value=""/>
```

Comment:
I see that the Backup Plugin uses `File_Storage_Provider` as the default ID, should we do that here as well?